csrc/executor.cpp

```diff
@@ -1941,17 +1941,19 @@ std::vector<at::Tensor> FusionExecutor::runFusion(
 
     if (measure_kernel_time) {
       kernel_time_ms_ = timer.elapsed();
+    }
+  }
 
-      outputBytesProcessed(outputs);
+  if (measure_kernel_time) {
+    outputBytesProcessed(outputs);
 
-      if (isDebugDumpEnabled(DebugDumpOption::EffectiveBandwidth) ||
-          isDebugDumpEnabled(DebugDumpOption::PerfDebugVerbose)) {
-        double gb_per_s =
-            ((double)bytesProcessed() / ((double)kernel_time_ms_ / 1000)) /
-            (double)1.0e9;
-        debug() << "kernel" << kernel_id_ << " run in " << kernel_time_ms_
-                << " ms, achieved: " << gb_per_s << " GB/s" << std::endl;
-      }
+    if (isDebugDumpEnabled(DebugDumpOption::EffectiveBandwidth) ||
+        isDebugDumpEnabled(DebugDumpOption::PerfDebugVerbose)) {
+      double gb_per_s =
+          ((double)bytesProcessed() / ((double)kernel_time_ms_ / 1000)) /
+          (double)1.0e9;
+      debug() << "kernel" << kernel_id_ << " run in " << kernel_time_ms_
+              << " ms, achieved: " << gb_per_s << " GB/s" << std::endl;
```

Comment:
This change means that we print bytes processed even when we do not execute the kernel, correct?

Reply:
Correct -- I do see `gb_per_s=inf` printed, which is the right answer. 

I believe this will change again with https://github.com/NVIDIA/Fuser/pull/1930. This printing will have to be skipped because `kernel_id_` won't even exist.